### PR TITLE
Update aws elb parser

### LINKED ISSUE #4188: AWS loadbalancer parser
**Description of problem:**

Make aws loadbalancer parser support network and classic loadbalancer logs.

**Source data:**

[Please provide the source data you used when you experienced the problem.
For publicly available data please provide a URL or path of the source data.](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html)

**Plaso version:**

For example 20220724

### PR DESCRIPTION
**Pull request by alexgoedeke**
## One line description of pull request

update aws elb parser to support classic and network loadbalancer logs

## Description:

**Related issue (if applicable):** fixes #4188

## Notes:
All contributions to Plaso undergo [code
review](https://github.com/log2timeline/l2tdocs/blob/main/process/Code%20review%20process.asciidoc). This makes sure
that the code has appropriate test coverage and conforms to the Plaso [style
guide](https://plaso.readthedocs.io/en/latest/sources/developer/Style-guide.html).

One of the maintainers will examine your code, and may request changes. Check off the items below in
order, and then a maintainer will review your code.

## Checklist:
  - [x] Automated checks (Travis, Codecov, Codefactor )pass
  - [x] No new [new dependencies](https://plaso.readthedocs.io/en/latest/sources/developer/Adding-a-new-dependency.html) are required or l2tdevtools has been updated
  - [x] Reviewer assigned

### CODE REVIEW

**Review by jleaniz — CHANGES_REQUESTED** [review]
@alexgoedeke Thank you for the parser enhancements! Some minor comments/questions but the changes look good.

**Review by jleaniz — APPROVED** [review]

File: plaso/parsers/aws_elb_access.py [hunk 1]
```diff
@@ -115,16 +175,15 @@ class AWSELBParser(text_parser.PyparsingSingleLineTextParser):
...
   _ENCODING = 'utf-8'
 
-  BLANK = pyparsing.Literal('"-"')
-
-  _WORD = pyparsing.Word(pyparsing.printables) | BLANK
+  BLANK = pyparsing.Literal('"-"') | pyparsing.Literal('-')
 
-  _QUOTE_INTEGER = (
-      pyparsing.OneOrMore('"') + text_parser.PyparsingConstants.INTEGER | BLANK)
+  _WORD = pyparsing.Word(pyparsing.printables) | BLANK | pyparsing.Literal('-')
```
Comment (joachimmetz) [inline plaso/parsers/aws_elb_access.py:180]: BLANK already includes `pyparsing.Literal('-')` ?

File: plaso/parsers/aws_elb_access.py [hunk 2]
```diff
@@ -191,7 +259,59 @@ class AWSELBParser(text_parser.PyparsingSingleLineTextParser):
...
+      _DESTINATION_IP_ADDRESS_PORT.setResultsName('destination_ip_port') +
+      _FLOAT.setResultsName('request_processing_time') +
+      _FLOAT.setResultsName('destination_processing_time') +
+      _FLOAT.setResultsName('response_processing_time') +
+      _INTEGER.setResultsName('elb_status_code') +
+      _INTEGER.setResultsName('destination_status_code') +
+      _INTEGER_NEGATIVE.setResultsName('received_bytes') +
+      _INTEGER_NEGATIVE.setResultsName('sent_bytes') +
+      pyparsing.quotedString.setResultsName('request')
+          .setParseAction(pyparsing.removeQuotes) +
```
Comment (joachimmetz) [inline plaso/parsers/aws_elb_access.py:301]: break line after .setParseAction(

File: plaso/parsers/aws_elb_access.py [hunk 3]
```diff
@@ -191,7 +259,59 @@ class AWSELBParser(text_parser.PyparsingSingleLineTextParser):
...
+      pyparsing.quotedString.setResultsName('user_agent')
+          .setParseAction(pyparsing.removeQuotes) +
+      _WORD.setResultsName('ssl_cipher') +
+      _WORD.setResultsName('ssl_protocol')
+  )
+
+  LINE_STRUCTURES = [
+    ('elb_application_accesslog', _LOG_LINE_APPLICATION),
```
Comment (joachimmetz) [inline plaso/parsers/aws_elb_access.py:309]: codebase uses 4 spaces for continuation indentation

File: plaso/parsers/aws_elb_access.py [hunk 4]
```diff
@@ -315,22 +438,49 @@ def ParseRecord(self, parser_mediator, key, structure):
...
-    elb_request_received_event = time_events.DateTimeValuesEvent(
-        date_time_request_received,
-        definitions.TIME_DESCRIPTION_AWS_ELB_REQUEST_RECEIVED)
-
     parser_mediator.ProduceEventWithEventData(
         elb_response_sent_event, event_data)
 
-    parser_mediator.ProduceEventWithEventData(
-        elb_request_received_event, event_data)
+    if key == 'elb_application_accesslog' and \
```
Comment (joachimmetz) [inline plaso/parsers/aws_elb_access.py:474]: don't use `\' to break up lines

File: tests/parsers/aws_elb_access.py [hunk 5]
```diff
@@ -32,6 +32,8 @@ def testParse(self):
     # we sort the events.
     events = list(storage_writer.GetSortedEvents())
 
+    # pylint: disable=line-too-long
```
Comment (joachimmetz) [inline tests/parsers/aws_elb_access.py:35]: don't do this

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**